src/capi.rs

```diff
@@ -1104,9 +1127,15 @@ pub unsafe extern fn rav1e_send_frame(
     (*frame).opaque.take().map(rav1e::Opaque::new)
   };
 
+  let t35_metadata = if frame.is_null() {
+    Box::new([])
+  } else {
+    (*frame).t35_metadata.clone().into_boxed_slice()
```

Comment:
alternatively [mem::take](https://doc.rust-lang.org/std/mem/fn.take.html) might be of use?

Reply:
seems OK as we hide the frame contents from the c api and there's no way to read back a frame after it's been sent, updated to use it